framework/DataObjects/HistorySet.py

```diff
@@ -114,10 +114,16 @@ def _updateSpecializedInputValue(self,name,value,options=None):
     # if this flag is true, we accept realizations in the input space that are not only scalar but can be 1-D arrays!
     #acceptArrayRealizations = False if options == None else options.get('acceptArrayRealizations',False)
     unstructuredInput = False
-    if (not isinstance(value,(float,int,bool,np.ndarray,c1darray))):
+    if isinstance(value,np.ndarray):
+      if value.shape == (): #can't cast single-entry ND array into a c1darray, so make it into a single entry
```

Comment:
i think we both missed that it also changes things besides the formatting.  Opps.